### PR TITLE
Feature/89/notify tab change in frame

### PR DESCRIPTION
**Pull request by MaartenWillemsAgilent**
Added broadcast of event when frame gets activated. The activated frame is send together with the event.

### CODE REVIEW

**Review by MichaelVyverman — APPROVED** [review]
LGTM

**Comment by MaartenWillemsAgilent** [comment]
> LGTM

I added a commit to delay the broadcast of the message until the fade in method is finished. The tabulator requires the frame to be fully initialised before it can be created. Only then it can correctly calculate the height.

File: Panels/Frame.js [hunk 1]
```diff
@@ -1365,14 +1365,15 @@ define([
...
-                frame._updateMemberVisibility();
+                frame._updateMemberVisibility( () => Msg.broadcast("frame_activated", frame.getActiveMemberFrame().getId()));
                 frame._getTabContainer().children('.AXMFrameTabContainerInner').children('.AXMFrameTabElement')
                     .removeClass('AXMFrameTabElementActive')
                     .addClass('AXMFrameTabElementInActive');
                 $('#'+frame._getTabId(fnr))
                     .removeClass('AXMFrameTabElementInActive')
                     .addClass('AXMFrameTabElementActive');
 
+                ;
```
Comment (MichaelVyverman) [inline Panels/Frame.js:1376]: Is this semicolon intended?

Comment (MaartenWillemsAgilent) [reply Panels/Frame.js:1376]: Nope :-s

**Pull request merged**